resources/qml/Menus/MaterialBrandMenu.qml

```diff
@@ -94,6 +94,8 @@ Cura.MenuItem
         // We have to keep a count of itemHovered (instead of just a bool)
         property int itemHovered: 0
 
+        implicitX: parent.width - UM.Theme.getSize("default_lining").width
```

Comment:
I imagine this value should be used in most cases, and as such should be de default value. Why not change the value in https://github.com/Ultimaker/Cura/blob/e99719a08fb32338aafaaa13646e483580bae7b3/resources/qml/Menus/MaterialBrandSubMenu.qml#L22 ?

Reply:
Makes sense 👍 I had to set implicitX: parent.width in the nested MaterialBrandMenu because they require different values to line up correctly.